Rakefile

```diff
@@ -1,9 +1,102 @@
 # Add your own tasks in files placed in lib/tasks ending in .rake,
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
```

Comment:
We've got a couple of other tasks in lib/tasks already, probably worth putting these new ones in a separate file there too

Reply:
:joy: Wow I can't believe I missed that comment. I'll move the tasks I've added.

Reply:
I've moved the new tasks to `lib/tasks/deploy.rake`